nixos/modules/programs/firefox.nix

```diff
@@ -224,11 +224,11 @@ in
         extraPrefs = cfg.autoConfig;
         extraNativeMessagingHosts =
           old.extraNativeMessagingHosts or []
-          ++ optional nmh.ff2mpv ff2mpv
-          ++ optional nmh.euwebid web-eid-app
+          ++ optional nmh.ff2mpv pkgs.ff2mpv
+          ++ optional nmh.euwebid pkgs.web-eid-app
           ++ optional nmh.gsconnect gnomeExtensions.gsconnect
```

Comment:
This should probably also be `pkgs.gnomeExtensions`

Reply:
I'm not sure why, but that one didn't seem to be a problem.  I can add it too though.